docs/data/material/getting-started/templates/templates.md

```diff
@@ -17,14 +17,12 @@ for reuse in other pages.
 For multi-part examples, a table in the README at the linked source code location describes
 the purpose of each file.
 
-{{"demo": "Templates.js", "hideToolbar": true, "bg": true}}
+{{"component": "docs/src/modules/components/MaterialFreeTemplatesCollection.js"}}
 
 If while using these examples you make changes or enhancements that could improve the
 developer experience, or you would like to contribute an additional example,
 please consider creating a [pull request on GitHub](https://github.com/mui/material-ui/pulls).
```

Comment:
This can definitely be condensed!

```suggestion
See any room for improvement?
Please feel free to open an [issue](https://github.com/mui/material-ui/issues/new/choose) or [pull request](https://github.com/mui/material-ui/pulls) on GitHub.
```